server/utils/cloudFunctions/functions/test/deleteCollection.js

```diff
@@ -0,0 +1,43 @@
+/**
+ * Description: code copied from here: `https://firebase.google.com/docs/firestore/manage-data/delete-data`
+ *              because firestore doesn't have a built in way to delete/reset
+ *              a (sub)collection. 
+ */
+
+const DEFAULT_BATCH_SIZE = 10;
+
+async function deleteCollection(
```

Comment:
Why is this in the test directory?

Reply:
It seemed appropriate to put it there because the only legitimate use case I could think of for it was for tearing down collections before/after tests.